d2bs/kolbot/libs/SoloLeveling/BuildFiles/amazon.JavazonBuild.js

```diff
@@ -0,0 +1,50 @@
+/*
+ *    @filename   amazon.JavazonBuild.js
+ *	  @author	  isid0re

```

Comment:
nit: inconsistent indentation on @author compared to other files (such as CubingOverrides.js)